menas/src/main/scala/za/co/absa/enceladus/menas/repositories/OozieRepository.scala

```diff
@@ -222,21 +223,14 @@ class OozieRepository @Autowired() (oozieClientRes: Either[OozieConfigurationExc
     })
   }
 
-  /**
-   * Get status of a running workflow
-   */
-  def getJobStatus(jobId: String): Future[WorkflowStatus] = {
-    getOozieClientWrap({ oozieClient: OozieClient =>
-      oozieClient.getJobInfo(jobId).getStatus
-    })
-  }
-
   /**
    * Kill a running coordinator
    */
   def killCoordinator(coordId: String): Future[Unit] = {
-    getOozieClientWrap({ oozieClient: OozieClient =>
-      oozieClient.kill(coordId)
+    getOozieClientWrap({ oozieClient =>
+      Try {
```

Comment:
Why add a `Try` here? `getOozieClientWrap` returns a Future, if an exception is thrown it will be caught by `Try` and swallowed. The `Future` will succeed without a trace.